app/Helpers/WikipediaHelper.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Helpers;
+
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Str;
+
+class WikipediaHelper
+{
+    /**
+     * Return the information about the given city or country from Wikipedia.
+     * All API calls are documented here:
+     * https://www.mediawiki.org/w/api.php?action=help&modules=query.
+     *
+     * @param  string  $topic
+     * @return array
+     */
+    public static function getInformation(string $topic): array
+    {
+        $query = http_build_query([
+            'action' => 'query',
+            'prop' => 'description|pageimages',
+            'titles' => $topic,
+            'pithumbsize' => 400,
+            'format' => 'json',
+        ]);
+
+        $url = 'https://en.wikipedia.org/w/api.php?'.$query;
```

Comment:
I don't think it's a good idea to hardcode the english wikipedia url.

Reply:
I agree. Although I'll do it in #425